oracle-sqlcl/helm/values.yaml

```diff
@@ -0,0 +1,62 @@
+image:
+  repository: quay.io/ecosystem-appeng/oracle-sqlcl-mcp-server
+  tag: "1.0.0"
```

Comment:
This should be derived from the chart version by default (see comment in template)